airbyte-cdk/bulk/core/load/src/integrationTest/kotlin/io/airbyte/cdk/load/mock_integration_test/MockBasicFunctionalityIntegrationTest.kt

```diff
@@ -250,4 +271,187 @@ class MockBasicFunctionalityIntegrationTest :
             returnedStateMessage,
         )
     }
+
```

Comment:
this is identical to the functions deleted from BasicFunctionalityIntegrationTest, no need to review